specification/communication/data-plane/Microsoft.CommunicationServicesAdministration/preview/2020-11-01-preview3/phonenumbers.json

```diff
@@ -0,0 +1,1014 @@
+      "delete": {
+        "tags": ["Manage acquired phone numbers"],
+        "operationId": "PhoneNumberAdministration_DeletePhoneNumber",
+        "summary": "Releases an acquired phone number.",
+        "produces": [
+          "application/json"
+        ],
+        "responses": {
+          "202": {
+            "description": "The service has accepted the request and will start processing later. It will return 'Accepted' immediately and include an 'Operation-Location' header. Client side should further query the operation status using the URL specified in this header.",
+            "headers": {
+              "Operation-Location": {
+                "description": "URL to query for status of the operation.",
+                "type": "string"
+              }
+            }
+          },
+          "default": {
+            "description": "Failure",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "CapabilityValue": {
+      "description": "",
+      "type": "string",
+      "enum": [
+        "inbound",
+        "outbound",
+        "inbound+outbound",
+        "none"
+      ],
+      "x-ms-enum": {
+        "name": "CapabilityValue",
+        "modelAsString": true
+      }
+    },
+    "Capabilities": {
+      "description": "",
+      "type": "object",
+      "properties": {
+        "sms": {
+          "description": "Available Sms capabilities.",
+          "$ref": "#/definitions/CapabilityValue"
+        },
+        "calling": {
+          "description": "Available Calling capabilities.",
+          "$ref": "#/definitions/CapabilityValue"
+        }
+      }
+    },
+    "AvailableCapabilities": {
+      "description": "",
+      "type": "object",
+      "properties": {
+        "sms": {
+          "description": "Available Sms capabilities.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/CapabilityValue"
+          }
+        },
+        "calling": {
+          "description": "Available Calling capabilities.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/CapabilityValue"
+          }
+        }
+      }
+    },
+    "PhoneNumberType": {
+      "description": "",
+      "type": "string",
+      "enum": [
+        "tollFree",
+        "geographic"
+      ],
+      "x-ms-enum": {
+        "name": "PhoneNumberType",
+        "modelAsString": true
+      }
+    },
+    "AcquiredPhoneNumber": {
+      "description": "",
+      "type": "object",
+      "properties": {
+        "id": {
+          "description": "The id, which is the phone number in E.164 format without the leading +.",
+          "type": "string"
+        },
+        "phoneNumber": {
+          "description": "The phoneNumber in E.164 format.",
+          "type": "string"
+        },
+        "country": {
+          "description": "The country the phone number belongs to.",
```

Comment:
Country Code should have the description "The ISO 3166-2 code the country the phone number belongs to"